lib/cronofy/client.rb

```diff
@@ -96,6 +96,8 @@ def list_calendars
     #                              with symbolized keys (optional):
     #                              :description - A String describing the
     #                                             location.
+    #                              :lat - A String of the location's latitude.
+    #                              :long - A String of the location's longitude.
```

Comment:
Do you want to update the example https://github.com/cronofy/cronofy-ruby/pull/24/files#diff-08814dc079b4842e0c4c960d17e97abeR122

Also worth adding to the specs as these serve as an example usage